src/Form/ProductOptionForm.php

```diff
@@ -93,6 +93,14 @@ public function form(array $form, FormStateInterface $form_state) {
       '#default_value' => $option->getElementType(),
     ];
 
+    $form['productTypes'] = [
+      '#type' => 'entity_autocomplete',
```

Comment:
Could you add the `@I` annotation as a future improvement to turn this into a select list.